lib/coursemology/evaluator/client.rb

```diff
@@ -31,7 +35,7 @@ def client_loop
       # :nocov:
       # This sleep might not be triggered in the specs, because interruptions to the thread is
       # nondeterministically run by the OS scheduler.
-      sleep(1.minute)
+      sleep(@poll_time.send(@poll_interval))
```

Comment:
You can stub out sleep, and check that the correct timeout is specified as a spec